rvs/conf/deviceid.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# To use this utlity call it from testcript with below example
+# Example:  ./conf/deviceid.sh "conf/gpup_*.conf"
+# Above command will add device_id : XXXX in all gpup_*.conf files
+ 
+conf_to_mod=(echo $1)
```

Comment:
instead of $1 , can you name the variable , assign $1 to a variable

Reply:
$1 is used to get command line arguments(passed .conf files to .sh). In case of argument gst_*.conf to check for all conf $1 assignment to a variable is converting for the first file from an array. So right now keeping it $1 and will modify once I will make it working with variable.